packages/sdk/src/swap/v2/types.ts

```diff
@@ -140,6 +142,9 @@ interface Sending extends Receiving {
 }
 
 type SwapState =
+  | ({
+      state: 'PENDING';
```

Comment:
not sure if pending is the best term here - what do you think about `READY`/`OPEN`/`CREATED`?

Reply:
or `REQUESTED`? bc the method to open the channel is called `requestDepositAddress`